siafile.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"gopkg.in/mgo.v2/bson"
+	"time"
+)
+
+type Siafile struct {
+	Id           bson.ObjectId `bson:"_id,omitempty", json:"_id"`
+	Title        string        `json:"title"`
+	Description  string        `json:"description"`
+	Filename     string        `json:"filename"`
+	Content      string        `json:"content"`
+	UploadedTime time.Time     `json:"uploadedTime"`
+	Tags         []string      `json:"tags"`
+}
```

Comment:
it's less convenient for us because our API structs aren't all defined in one place. We would have to add json tags to every object that gets returned by the API, which are scattered across the codebase.